website/content/docs/commands/audit/enable.mdx

```diff
@@ -40,3 +37,9 @@ flags](/vault/docs/commands) included on all commands.
 - `-path` `(string: "")` - Place where the audit device will be accessible. This
   must be unique across all audit devices. This defaults to the "type" of the
   audit device.
+
+There are a set of [common options](/vault/docs/audit#common-configuration-options)
+which can be applied to all types of audit device.
+
+For more details, please see the relevant page for an audit device `type`, under
+[Audit Devices docs](/vault/docs/audit).
```

Comment:
```suggestion
All audit devices support a
[common set of options](/vault/docs/audit#common-configuration-options), for
more details, refer to the specific audit device type in the
[Audit Devices](/vault/docs/audit) overview.
```

Style correction: write in active voice, simplify sentence structure when possible